coreutils/Makefile

```diff
@@ -467,3 +470,53 @@ csv-exp-201609:
 		done ; \
 	done
 
+# This target is used for running coreutils experiment with LLBMC.
+%.llbmc :
+	if [ $* = "dd" ]; then \
+		PROGRAM_ARGS="xxxxxxxxxx xxxxxxxxxx xxxxxxxxxx" ; \
+	elif [ $* = "dircolors" ]; then \
+		PROGRAM_ARGS="xxxxxxxxxx xxxxxxxxxx xxxxxxxxxx" ; \
+	elif [ $* = "echo" ]; then \
+		PROGRAM_ARGS="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" ; \
+	elif [ $* = "expr" ]; then \
+		PROGRAM_ARGS="xxx xxx xxxxxxxxxx" ; \
+	elif [ $* = "mknod" ]; then \
+		PROGRAM_ARGS="xxx xxx xxxxxxxxxx" ; \
+	elif [ $* = "od" ]; then \
+		PROGRAM_ARGS="xxxxxxxxxx xxxxxxxxxx xxxxxxxxxx" ; \
+	elif [ $* = "pathchk" ]; then \
+		PROGRAM_ARGS="x x" ; \
+	elif [ $* = "printf" ]; then \
+		PROGRAM_ARGS="xxxxxxxxxx xxxxxxxxxx xxxxxxxxxx" ; \
+	else \
+		PROGRAM_ARGS="xx xx xxxxxxxxxx" ; \
+	fi ; \
+	if [ ! -d ${LLBMC_OUTPUT_DIR} ]; then \
+		mkdir ${LLBMC_OUTPUT_DIR}; \
+	fi ; \
+	${CC} -emit-llvm -c ${CFLAGS_LLBMC_COREUTILS} ${COREUTILS_SRC}/$*.c -o ${COREUTILS_SRC}/$*.bc ; \
+	llbmc ${LLBMC_OPTIONS} -arguments="$$PROGRAM_ARGS" ${COREUTILS_SRC}/$*.bc > ${LLBMC_OUTPUT_DIR}/$* ; \
+	rm -rf ${COREUTILS_SRC}/$*.bc ; \
+
+# This target is used for collecting data from coreutils LLBMC experiment into log-llbmc file.
+llbmc-csv:
+	rm -f ${LLBMC_OUTPUT_DIR}/log-llbmc.csv ; \
+	echo "Program, Time of Performing transformations on LLVM IR,Time of Performing logical encoding,Time of Performing formula transformations,Time of Running SMT solver (STP with MiniSat), Total Time,Error Status" >> ${LLBMC_OUTPUT_DIR}/log-llbmc.csv; \
```

Comment:
Please change "Time of Performing" to "Time for performing". Please note the capitalization.